README.md

```diff
@@ -113,12 +111,14 @@ Within the container, build the font:
     $ make -C src/icons
 
 <!--- reference links for badges -->
-[all-contrib]: https://img.shields.io/badge/all_contributors-125-orange.svg?style=flat-square "All Contributors badge"
+[all-contrib]: https://img.shields.io/badge/all_contributors-125-orange.svg "All Contributors badge"
 [build-status-badge]: https://travis-ci.org/ForkAwesome/Fork-Awesome.svg?branch=master "Build status badge"
 [build-status-link]: https://travis-ci.org/ForkAwesome/Fork-Awesome
-[cdnjs-badge]: https://img.shields.io/cdnjs/v/fork-awesome.svg?style=flat-square "CDNJS badge"
+[cdnjs-badge]: https://img.shields.io/cdnjs/v/fork-awesome.svg "CDNJS badge"
 [cdnjs-link]: https://cdnjs.com/libraries/fork-awesome
-[jsdeliver-badge]: https://data.jsdelivr.com/v1/package/npm/fork-awesome/badge "JSDeliver badge"
+[jsdeliver-badge]: https://data.jsdelivr.com/v1/package/npm/fork-awesome/badge?style=rounded "JSDeliver badge"
```

Comment:
```suggestion
[jsdeliver-badge]: https:/img.sheilds.io/jsdelivr/npm/hm/fork-awesome "JSDeliver badge"
```

For the sake of consistency, since we use shields.io everywhere else.

( PS : I personally prefer flat badges )

Reply:
I think that should be img.sh**ie**lds.io... Otherwise cool.

I slightly prefer rounded, but I don't mind, just want them all the same style

Reply:
The feeling when you realize that you messed up a suggestion for a fix :facepalm:

fixed the suggestion :see_no_evil:

Reply:
This suggestion was messed up from the start. The fix to the first mess up still did not fix it. There was a `/` missing after the `https:/` :facepalm: